package.json

```diff
@@ -30,15 +46,15 @@
 		"js-yaml": "^4.1.0",
 		"pcre-to-regexp": "^1.1.0",
 		"zod": "^3.21.4",
-		"zodcli": "^0.0.4"
+		"zodcli": "^0.0.4",
+		"@cloudflare/workers-types": "^4.20230404.0"
```

Comment:
Is it normal to move to proper deps when exporting types? I'm not familiar with the common practices here.

Reply:
yeah I am not too familiar with the common practices either, but I thought that the consumer of next-on-pages will need access to the types, so this package is needed as a standard dependency so that they will get them

but I don't think I've tried to see if it works with the package set as a dev dependency, I'll check that and will let you know, thanks :smiley:

Reply:
Ok, I've tried putting this back (see https://github.com/cloudflare/next-on-pages/pull/265/commits/0e7f67a291d9c8411ff34d6760827d002adb1355, [prerelease](https://prerelease-registry.devprod.cloudflare.dev/next-on-pages/runs/5031931405/npm-package-next-on-pages-265)) but as I expected it then stops working:

![Screenshot 2023-05-20 at 12 34 04](https://github.com/cloudflare/next-on-pages/assets/61631103/979d3bd9-f62b-4512-95b9-f3aaf1eb25ee)

![Screenshot 2023-05-20 at 12 33 50](https://github.com/cloudflare/next-on-pages/assets/61631103/bb079b7b-bf1d-4660-a558-4b601d72b4ba)

Reply:
vs having is as a dependency (see https://github.com/cloudflare/next-on-pages/pull/265/commits/dd3e1ce138542fb674bd8e371b2f818118b3d1b2, [prerelease](https://prerelease-registry.devprod.cloudflare.dev/next-on-pages/runs/5031974631/npm-package-next-on-pages-265)):
![Screenshot 2023-05-20 at 12 48 37](https://github.com/cloudflare/next-on-pages/assets/61631103/4fc16002-85c0-4423-9cba-abf6387b1738)

![Screenshot 2023-05-20 at 12 48 55](https://github.com/cloudflare/next-on-pages/assets/61631103/41f30388-01cb-45e4-bb86-0e8f6f02b897)

Reply:
Cool 👍